doc/api/crypto.md

```diff
@@ -2631,7 +2631,7 @@ changes:
                  `ERR_INVALID_CALLBACK`.
 -->
 
-* `size` {number}
+* `size` {number} The `size` must not be larger than `2**31 - 1`.
```

Comment:
Nit: perhaps to avoid negation in the sentence:
```suggestion
* `size` {number} The `size` must be less or equal to `2**31 - 1`.
```
(same applies to other places)